config.json

```diff
@@ -18,6 +18,20 @@
       ]
     },
     {
+      "core": false,
+      "difficulty": 1,
```

Comment:
I think the difficulty should be 3 here.

(The difficulty ratings right now are purely me taking a swag guess a while back, and I don't think I've had any input ever on whether they're wildly off. So we're just guessing here).

Reply:
You're right, isogram is certainly not in the same ballpark as the hello world exercise. I made it a 3 now.